### PR TITLE
Validation for user namespace in the config.

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by LK4D4** [comment]
Nice. Shouldn't we also check that there is no mappings in config too? Like we check for network namespace.

**Comment by mrunalp** [comment]
@avagin @LK4D4 updated, thanks.

**Comment by avagin** [comment]
LGTM, thanks

File: configs/validate/config.go [hunk 1]
```diff
@@ -74,3 +78,16 @@ func (v *ConfigValidator) security(config *configs.Config) error {
 	}
 	return nil
 }
+
+func (v *ConfigValidator) usernamespace(config *configs.Config) error {
+	if config.Namespaces.Contains(configs.NEWUSER) {
+		if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
```
Comment (dqminh) [inline configs/validate/config.go]: should we check if there's any other errors here too i.e. `err != nil` ?

Comment (mrunalp) [inline configs/validate/config.go]: I think it should be okay not to check for anything else here.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**